presentation/src/main/java/com/architect/coders/mu8/di/AppModule.kt

```diff
@@ -0,0 +1,14 @@
+package com.architect.coders.mu8.di
+
+import com.architect.coders.mu8.data.DataApp
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+@Module
+class AppModule {
```

Comment:
@jamacado can we remove this Module? Is not being used at the moment.

Reply:
ok, remove